modules/tracking/samples/lasot_benchmark.py

```diff
@@ -0,0 +1,153 @@
+
+        if tracker_name == "Boosting":
+            tracker = cv.TrackerBoosting_create()
+        elif tracker_name == "MIL":
+            tracker = cv.TrackerMIL_create()
+        elif tracker_name == "KCF":
+            tracker = cv.TrackerKCF_create()
+        elif tracker_name == "MedianFlow":
+            tracker = cv.TrackerMedianFlow_create()
+        elif tracker_name == "MOSSE":
+            tracker = cv.TrackerMOSSE_create()
+        elif tracker_name == "CSRT":
+            tracker = cv.TrackerCSRT_create()
+        elif tracker_name == "GOTURN":
+            tracker = cv.TrackerGOTURN_create()
+
+        print("Tracker: ", tracker)
+
+        init_once = False
+
+        print("Video: " + str(video_name))
+
+        true_positive = 0
+        false_positive = 0
+        false_negative = 0
+
+        video_name = video_name.replace("\n", "")
+        ground_truth = open(args.path_to_dataset +
+                            video_name + "/groundtruth.txt", "r")
+        ground_truth_bb = ground_truth.readline().replace("\n", "").split(",")
+        init_bb = ground_truth_bb
+
+        for _ in range(len(init_bb)):
+            init_bb[_] = float(init_bb[_])
+
+        init_bb = tuple(init_bb)
+
+        video_sequence = sorted(
+            glob.glob(str(args.path_to_dataset + str(video_name) + "/img/*.jpg")))
+
+        print("Number of frames: " + str(len(video_sequence)))
+        # Loop for frames
+        for f, image in enumerate(video_sequence):
+
+            for _ in range(len(ground_truth_bb)):
+                ground_truth_bb[_] = float(ground_truth_bb[_])
+
+            ground_truth_bb = tuple(ground_truth_bb)
+
+            frame = cv.imread(image)
+
+            if ((f + 1) % 100 == 0) and (f != 0):
+                if tracker_name == "Boosting":
+                    tracker = cv.TrackerBoosting_create()
+                elif tracker_name == "MIL":
+                    tracker = cv.TrackerMIL_create()
+                elif tracker_name == "KCF":
+                    tracker = cv.TrackerKCF_create()
+                elif tracker_name == "MedianFlow":
+                    tracker = cv.TrackerMedianFlow_create()
+                elif tracker_name == "GOTURN":
+                    tracker = cv.TrackerGOTURN_create()
+                elif tracker_name == "MOSSE":
+                    tracker = cv.TrackerMOSSE_create()
+                elif tracker_name == "CSRT":
+                    tracker = cv.TrackerCSRT_create()
+                init_once = False
+                init_bb = new_bb
+
+            if not init_once:
+                temp = tracker.init(frame, init_bb)
+                init_once = True
+
+            temp, new_bb = tracker.update(frame)
+            # Coordinates of points of bounding boxes
+            new_bb_xmin = new_bb[0] - new_bb[2] / 2
+            new_bb_xmax = new_bb[0] + new_bb[2] / 2
+            new_bb_ymin = new_bb[1] - new_bb[3] / 2
+            new_bb_ymax = new_bb[1] + new_bb[3] / 2
```

Comment:
`update` should return `x, y, w, h` where `x, y` are left top points

Reply:
I correct formulas for evaluation. Also I fix case if ground truth bounding box is (0, 0, 0, 0).